apps/web/cypress/tests/integrations-list-page.spec.ts

```diff
@@ -89,6 +122,28 @@ describe('Integrations List Page', function () {
 
     cy.wait('@getIntegrations');
 
+    checkTableRow(
+      {
+        name: 'Novu Email',
+        key: null,
+        provider: 'Novu Email',
+        channel: 'Email',
+        environment: null,
+        status: 'Disabled',
+      },
+      0
+    );
+    checkTableRow(
+      {
+        name: 'Novu SMS',
+        key: null,
+        provider: 'Novu SMS',
+        channel: 'SMS',
+        environment: null,
+        status: 'Disabled',
+      },
```

Comment:
```suggestion
    checkTableRow(
      {
        name: 'Novu Email',
        provider: 'Novu Email',
        channel: 'Email',
        status: 'Disabled',
      },
      0
    );
    checkTableRow(
      {
        name: 'Novu SMS',
        provider: 'Novu SMS',
        channel: 'SMS',
        status: 'Disabled',
      },
```
Based on my previous suggestions.